base/src/main/res/values-de-rDE/strings.xml

```diff
@@ -44,15 +44,15 @@
     <string name="skip">Überspringen</string>
     <string name="exportButton">Exportieren</string>
     <string name="importButton">Importieren</string>
-    <string name="export_to">Exportieren Sie nachverfolgte Daten in dieser Gruppe in CSV:</string>
-    <string name="import_from">Nachverfolgte Daten aus CSV in diese Gruppe importieren:</string>
+    <string name="export_to">Exportieren Sie nachverfolgte Daten in dieser Gruppe nach CSV:</string>
+    <string name="import_from">Importieren Sie nachverfolgte Daten von CSV in diese Gruppe:</string>
     <string name="import_warning">Achtung: Der Import ist irreversibel. Track &amp; Graph versucht, alle importierten Daten mit vorhandenen Daten in dieser Gruppe nach Namen zusammenzuführen.</string>
     <string name="graphs_and_statistics"><![CDATA[Diagramme & Statistiken]]></string>
     <string name="add_a_graph_or_stat">Ein Diagramm oder eine Statistik hinzufügen</string>
     <string name="graph_type_label">Diagrammtyp:</string>
-    <string name="sample_size">Größe des Samples:</string>
-    <string name="select_a_discrete_feature">Wählen Sie einen Datensatz mit Labels aus: </string>
-    <string name="no_labels">Dieser Datensatz hat keine Labels</string>
+    <string name="sample_size">Größe der Stichprobe:</string>
+    <string name="select_a_discrete_feature">Wählen Sie einen Datensatz mit Beschreibung aus: </string>
+    <string name="no_labels">Dieser Datensatz hat keine Beschriftung</string>
```

Comment:
Google translates this to: "This record has no caption" .. I just want to double check this means what you think it does. This is used if you select a feature (a data set/tracker) that contains all data points that have no labels.

Reply:
In English it is "This data set has no labels". It is a data set, many data points, not a single record, and specifically no labels, not notes or any thing else.

Reply:
You're right that "Beschriftung" doesn't really get the meaning, unfortunately neither do the other german translations used previously. Then I think it would be best to just use "Label" consistently. It should be common enough in german to be understood well. I'll add those changes.

Reply:
Regarding "Datensatz": That is a quite literal translation of "data set" (see e.g. https://en.pons.com/translate/english-german/data+set?bidir=1) and should capture that meaning well (it can also apply to multiple data points, not only single records).